service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/AdditionalApplicationsUploadedEventHandler.java

```diff
@@ -145,17 +149,27 @@ public void notifyApplicant(final AdditionalApplicationsUploadedEvent event) {
         final CaseData caseData = event.getCaseData();
         final OrderApplicant applicant = event.getApplicant();
 
+        AdditionalApplicationsBundle newBundleUploaded = caseData.getAdditionalApplicationsBundle().get(0).getValue();
+
         // DFPL-498 notify all LAs
-        final Set<String> recipients = new HashSet<>(localAuthorityRecipients.getRecipients(RecipientsRequest.builder()
-            .caseData(caseData).build()));
+        final Set<String> recipients = new HashSet<>();
 
-        if (applicant.getType() != LOCAL_AUTHORITY && applicant.getType() != SECONDARY_LOCAL_AUTHORITY) {
-            final Map<String, String> emails = getRespondentAndChildEmails(caseData);
-            if (isNotEmpty(emails.get(applicant.getName()))) {
-                recipients.add(emails.get(applicant.getName()));
+        if (newBundleUploaded.getC2DocumentBundle() != null || newBundleUploaded.getOtherApplicationsBundle() != null) {
+            recipients.addAll(localAuthorityRecipients.getRecipients(RecipientsRequest.builder()
+                .caseData(caseData).build()));
+
+            if (applicant.getType() != LOCAL_AUTHORITY && applicant.getType() != SECONDARY_LOCAL_AUTHORITY) {
```

Comment:
Simplify it as `List.of(LOCAL_AUTHORITY, SECOND_LOCAL_AUTHORITY).contains(applicant.getType())`

Reply:
Oh, that's existing code, but happy to change it 😎

Reply:
It is existing code, but i am happy to change it